File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        vector<int> temp;
        int n = nums.size();
        if(n == 0) return 0;
        temp.push_back(nums[0]);

        for(int i = 1; i<n; i++){
            if(nums[i] != nums[i-1]){
                temp.push_back(nums[i]);
            }
        }
        for(int i = 0; i  < temp.size(); i++){
            nums[i] = temp[i];
        }
        return temp.size();
        
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n= len(nums)
        if n == 0:
            return 0
        temp = []
        temp.append(nums[0])
        for i in range(1,n):
            if nums[i] != nums[i-1]:
                temp.append(nums[i])
        for i in range(len(temp)):
            nums[i] = temp[i]
        return len(temp)               
        